docs/docs/faq.md

```diff
@@ -125,23 +120,21 @@ We welcome contributions from folks interested in developing Icechunk bindings f
 
 ## Is Icechunk stable?
 
-The Icechunk library is reasonably well-tested and performant.
-The Rust-based core library provides a solid foundation of correctness, safety, and speed.
+Yes! Icechunk 1.0, released in July 2025, is a stable release suitable for production use.
+Data written by Icechunk 1.0 and greater will forever be readable by future Icechunk versions.
 
-However, the actual on disk format is still evolving and may change from one alpha release to the next.
-Until Icechunk reaches v1.0, we can't commit to long-term stability of the on-disk format.
-This means Icechunk can't yet be used for production uses which require long-term persistence of data.
+## What is the backwards compatibility policy for the Icechunk format?
 
-😅 Don't worry! We are working as fast as we can and aim to release v1.0 soon!
+Any data written by Icechunk v1.0 or great will be readable forever by future Icechunk release (backwards compatible).
```

Comment:
s/great/greater

But is greater the right word here? newer?

Reply:
"greater" is the word we want, yes. Greater / lesser is the way we compare numerical values.